### PR TITLE
Enable cross_entropy operator for a ngraph engine

### PR DESCRIPTION
**Pull request by mozga-intel**
Introduces the nGraph's implementation of the cross_entropy operator.
test=develop

### CODE REVIEW

**Comment by tensor-tang** [comment]
Please resolve https://github.com/PaddlePaddle/Paddle/pull/15375#discussion_r256247259 in this PR

**Comment by mozga-intel** [comment]
@luotao1, @tensor-tang Can you please continue review and comment this pull-request?. These operators are crucial for us.

File: python/paddle/fluid/tests/unittests/ngraph/test_cross_entropy_ngraph_op.py [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+    """
+
+    def init_x(self):
+        self.shape = [10, 2, 4]
+        self.ins_num = np.prod(np.array(self.shape))
+        self.X_2d = randomize_probability(self.ins_num,
+                                          self.class_num).astype(self.dtype)
+        self.x = self.X_2d.reshape(self.shape + [self.class_num])
+
+    def init_label(self):
```
Comment (tensor-tang) [inline python/paddle/fluid/tests/unittests/ngraph/test_cross_entropy_ngraph_op.py:149]: The tests below could be more clear and clean, you can polish them later.

**Review by tensor-tang — APPROVED** [review]
LGTM

**Pull request merged**